.github/workflows/ci.yml

```diff
@@ -67,6 +70,33 @@ jobs:
 
       - name: Run examples
         run: scripts/examples
-      
+
       - name: Check formatting
         run: cargo fmt --all -- --check
+
+  bazel-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - uses: bazelbuild/setup-bazelisk@v3
+
+      - name: Setup Bazel cache
+        uses: actions/cache@v4
+        with:
+          path: "~/.cache/bazel"
+          key: bazel
+
+      - name: Install libunwind
+        run: sudo apt-get install -y libunwind-dev
+
+      - name: Bazel build
+        run: |
+          git clone --depth 1 -b evm-rpc-canister https://github.com/dfinity/ic.git ../ic &&\
+          cd ../ic &&\
+          ln -s ../../../evm-rpc-canister rs/ethereum/evm_rpc &&\
+          sed -i 's/"threadpool"/"tiny-keccak": crate.spec(version = "2.0.2", features = ["keccak"]), "threadpool"/g' bazel/external_crates.bzl && CARGO_BAZEL_REPIN=1 bazel sync --only=crate_index &&\
```

Comment:
Temporary patch equivalent to [!19052](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/19052). Once that MR is merged, we can remove this line.